### ISSUE TITLE
Support for ranges?

### ISSUE THREAD

**Issue by slyrus** [a1b10/cl-xlsx#3]
It would be great if I could either specify a range to be read, or know the location of returned values. As things stand now, empty rows/columns aren't read so I have no idea where I am in my excel file. Thanks!

**Comment by gwangjinkim** [comment]
Yes. This package doesn't claim - yet - to be a real excel reader. It is more a hack.
I wrote it for my own use (simplest kind of tables - not really formatted - of large bioinformatics tables).
A real  Excel reader would be able to create a workbook and create a complex object.

I'm sorry that this is in such an initial state.

**Comment by gwangjinkim** [comment]
I might tackle this soon.
I am trying to translate an excel reader/writer Racket package at the moment. It will still take a lot of time. But a solution is on the way.

**Comment by gwangjinkim** [comment]
The new package gwangjinkim/cl-excel supports ranges. Please check it out!
